advanced/Templates/gravity.db.schema

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE whitelist (domain TEXT UNIQUE NOT NULL, disabled BOOLEAN DEFAULT 0, comment TEXT, DateAdded DATETIME);
```

Comment:
No, the reason for this is somewhat non-intuitive, so let me explain it here.

We use `.import gravity.list` to mass-import the domains into the database. This is be far the fastest way of doing it. The issue is that it will fill all unspecified fields with `NULL`, ignoring the `DEFAULT` option we set there.

Later on, we select the blacklists using
```
WHERE a.domain NOT IN (SELECT domain from whitelist WHERE disabled != 1);
```
This `disabled != 1` matches two things:
- `0` (really enabled), **and**
- `NULL` (never set).

If we would specify an `enabled` field, we would need to test for two things here
```
WHERE a.domain NOT IN (SELECT domain from whitelist WHERE enabled == 1 OR enabled IS NULL);
```

However, I see that this is probably more straightforward and will implement this with a positive (`enabled`) logic.